app/services/AuthService.ts

```diff
@@ -67,4 +72,99 @@ export const AuthService = {
 
     return payload;
   },
+
+  /**
+   *
+   *
+   * @param {String} token Authorization token attached to the HTTP header
+   * @return {users} Decoded jsonwebtoken
+   * @throws {JwtMalformedException} Throws error when token is malformed or empty
+   */
+
+  //will Sign and validate JWT token
+  async jwtSign(uid: string): Promise<string> {
+    const jwtAll = jwt.sign(
+      {
+        //will sign token for up to 24 hours
+        exp: Math.floor(Date.now() / 1000) + 24 * 60 * 60,
+        uid: uid,
+      },
+      config.app.secret
+    );
+
+    const found = await prisma.users.findFirst({
+      where: {
+        uid: uid,
+      },
+    });
+    if (found === null) {
+      // will create JWT token if user id is not found
```

Comment:
The comment here is not correct. It doesn't create jwt token but creates a row inside the database